keyboards/lets_split/keymaps/tylerwince/rules.mk

```diff
@@ -0,0 +1,6 @@
+MOUSEKEY_ENABLE=yes
+RGBLIGHT_ENABLE=yes
+TAP_DANCE_ENABLE=yes
+ifndef QUANTUM_DIR
```

Comment:
Lines 4-6 are leftover from a previous build system and no longer have any functionality. They should be deleted.

Reply:
@noroadsleft - Thanks so much for the review. I am a bit confused on this one. I use `RGBLIGHT` and `TAP_DANCE` in my keymap but are you saying that including them here doesn't matter and I should only have this file include the Makefile from the top root? (Effectively making this file only have:
```make
include ../../../../Makefile
```

Reply:
Dur -- read this wrong. 4-6. I should only include the features I use. Got it. :+1: